Source/AwsCommonRuntimeKit/crt/ByteBuffer.swift

```diff
@@ -30,6 +33,11 @@ public class ByteBuffer {
         currentEndianness = endianness
         return self
     }
+    
+    public func putBytes(_ value: UInt8) {
```

Comment:
Should this be a byte array? put(UInt8) seems to have this functionality covered already?

Reply:
I just needed another function that put a byte that didnt return self. want me to call it just `putByte` maybe?

Reply:
yeah, let's do `putByte`, `putBytes` is a LIE